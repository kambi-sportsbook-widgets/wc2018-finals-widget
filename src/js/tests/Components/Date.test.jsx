import React from 'react'
import { translationModule } from 'kambi-widget-core-library'
import ReactShallowRenderer from 'react-test-renderer/shallow'
import DateComponent from '../../Components/Date'

let renderer

const exampleDate = 1528988400000

jest.mock('kambi-widget-core-library', () => ({
  translationModule: {
    getTranslation: jest.fn(key => `Translated ${key}`),
  },
}))

describe('Date DOM rendering', () => {
  beforeEach(() => {
    renderer = new ReactShallowRenderer()
    translationModule.getTranslation = jest.fn(key => `Translated ${key}`)
  })

  it('renders correctly with example props', () => {
    expect(
      renderer.render(<DateComponent date={exampleDate} />)
    ).toMatchSnapshot()
  })
})
