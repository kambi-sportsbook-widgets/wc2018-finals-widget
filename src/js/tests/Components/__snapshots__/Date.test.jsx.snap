// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Date DOM rendering renders correctly with example props 1`] = `
<p>
  Translated day4 14 Translated month5 17:00
</p>
`;
